lib/vtls/openssl.c

```diff
@@ -2467,6 +2468,12 @@ static CURLcode ossl_connect_step1(struct connectdata *conn, int sockindex)
   }
 #endif
 
+#ifdef HAVE_SSL_CTX_SET_POST_HANDSHAKE_AUTH
+   /* OpenSSL 1.1.1 requires clients to opt-in for PHA */
+   SSL_CTX_set_post_handshake_auth(BACKEND->ctx, 1);
+   infof(data, "TLS 1.3 post-handshake auth\n");
```

Comment:
I don't think this infof() call will bring much happiness though, as it only enables auth so thus will always be shown with openssl 1.1.1....